scrapers_next/tn/committees.py

```diff
@@ -0,0 +1,44 @@
+from spatula import HtmlPage, HtmlListPage, CSS, XPath, SelectorError, SkipItem, URL, ListPage
```

Comment:
Remove unused imported classes

Reply:
For some reason the codespace won't let me resolve those issues. But I fixed the import and example_source lines so this is good to go!